### DISCUSSION TITLE
Using InertiaJS

### DISCUSSION THREAD

**Discussion by sayajin101** [Kyon147/laravel-shopify#154] [Q&A]
Hi there, I'm new to the whole Shopify App world.

I would like to know what is the best way to manage the session & route tokens using InertiaJS & VueJS 3.2 with script setup.
The current way I'm doing it by passing the shop name back and forward between the controller &  the .vue files feels extremely dirty.

Please any help will be appreciated

**Comment by StaffCollab** [comment]
@sayajin101 
I started just now as well:)
Have you discovered the golden rule?
